app/pb_kits/playbook/pb_radio/_radio.jsx

```diff
@@ -11,7 +11,7 @@ type RadioProps = {
   label: String,
   name: String,
   value: String,
-  checked?: Boolean,
+  defaultChecked?: Boolean,
```

Comment:
Is this solely for the case where the docs are being shown? If so, I'm not really for this change. We should implement a thin state wrapper around the doc to do this instead of modifying the kit itself unless I am missing some other requirement. https://reactjs.org/docs/uncontrolled-components.html#default-values

Reply:
@thestephenmarshall when I implemented the Block kit I noticed that there was an inversion originally made on the checked prop. So any block kit would be checked if defaultChecked was true. So my change was to use change the prop checked to defaultChecked. If anything I can revert this and implement a different solution.

Reply:
Can we not recode this to keep the checked prop...but fix the broken logic? Does the rails version use default_checked as a prop or checked?

Reply:
> So any block kit wouldn't be checked if defaultChecked was true

Do you mean within the context of `Block`'s documentation or the actual kit?